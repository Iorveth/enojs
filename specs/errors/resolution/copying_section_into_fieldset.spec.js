const eno = require('../../../eno.js');

const input = `
# original

# other

copy < original
entry = value
`.trim();

describe('resolution.copyingSectionIntoFieldset', () => {

  let error;

  beforeAll(() => {
    try {
      eno.parse(input);
    } catch(err) {
      error = err;
    }
  })

  it(`provides the correct message`, () => {
    expect(error.message).toMatchSnapshot();
  });

  it(`provides correct selection metadata`, () => {
    expect(error.selection).toMatchSnapshot();
  });
});
