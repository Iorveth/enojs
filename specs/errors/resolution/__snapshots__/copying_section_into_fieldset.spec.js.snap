// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolution.copyingSectionIntoFieldset provides correct selection metadata 1`] = `
Array [
  Array [
    4,
    0,
  ],
  Array [
    4,
    15,
  ],
]
`;

exports[`resolution.copyingSectionIntoFieldset provides the correct message 1`] = `
"In line 5 a section is copied into a fieldset.

   Line | Content
   ...
      3 | # other
      4 | 
 >    5 | copy < original
      6 | entry = value
"
`;
